garrysmod/lua/includes/modules/team.lua

```diff
@@ -146,6 +152,20 @@ function GetPlayers(index)
 
 end
 
+function GetAlivePlayers(index)
+
+	local TeamAlivePlayers = {}
+
+	for id,pl in ipairs( player.GetAll() ) do
+		if (IsValid(pl) and pl:Team() == index and pl:Alive()) then

```

Comment:
- `IsValid` check is unnecessary with `player/ents.GetAll`.
- [Code formatting isn't respected](https://github.com/Facepunch/garrysmod/blob/master/CONTRIBUTING.md#code-formatting).